utils/schemas.ts

```diff
@@ -102,9 +102,31 @@ export const validatedVendoJourneySchema = z.object({
 export type VendoJourney = z.infer<typeof vendoJourneySchema>;
 export type ValidatedVendoJourney = z.infer<typeof validatedVendoJourneySchema>;
 
-export const vbidSchema = z.object({
+export const VerbindungResponseSchema = z.object({
```

Comment:
This schema has nothing to do with the Bahn's VBIDs.

It is about the HTTP response from their API at /web/api/angebote/verbindung/<vbid>.

The VBID is just the sole parameter of the route, but the schema should be named after the route itself acting as a descriptor.